### PR TITLE
Removed white box from "cool services"

### PR DESCRIPTION
**Pull request by edricx**
Removed unnecessary white box around the contents of the "cool services" section in https://www.ocf.berkeley.edu/about/staff.

@sahilhasan

### CODE REVIEW

File: ocfweb/static/scss/pages/about/_about-staff.scss [hunk 1]
```diff
@@ -207,8 +207,8 @@ $color-blue3: #29D9C2;
         }
 
         > div > div > div {
-            background-color: $color-white;
-            color: black;
+            /* background-color: $color-white; */
```
Comment (matthew-mcallister) [inline ocfweb/static/scss/pages/about/_about-staff.scss]: Don't comment this out, just delete it.

**Comment by abizer** [comment]
I feel like the white on (green?) is a bit hard to read but otherwise lgtm

**Comment by edricx** [comment]
@abizer I changed the text to white since the header sentence ("OCF staff help provide...") was white originally, but that entire section can be changed to black text. Might actually make more sense because each of the colored sections on the page would then alternate between black and white text.

I was thinking of also getting rid of the white box around OCF Staff Meetings, but since its in some kind of special container the alignment with the left column would look weird. Left as is.

**Comment by matthew-mcallister** [comment]
Please change the text to black. White text on this background is just plain [hard to read](https://leaverou.github.io/contrast-ratio/#white-on-%2329D9C2).

**Comment by chriskuehl** [comment]
...but it looks better as it is now? :(

**Comment by chriskuehl** [comment]
can we just match the white box colors if that's the concern? this was put together in like one hour at 2am one night before a recruiting event, it doesn't surprise me things are a little mismatched.

but i really think it looks better now than with these changes.

**Comment by edricx** [comment]
All the text can be changed to black in the "cool services" section for better readability: https://leaverou.github.io/contrast-ratio/#black-on-%2329D9C2. 

Personally I think the original white box is out of place. I don't see why the three subsections in the box need to be separated as ideas/elements distinct from the rest of the section, so I don't think they should be separated visually either by putting them in a box.

**Comment by chriskuehl** [comment]
@edricx please post screenshots when you make these kinds of changes in PRs

**Comment by matthew-mcallister** [comment]
Simulated:

screenshot_2017-11-28_12-39-21

I fail to see how this is an improvement...

**Comment by edricx** [comment]
I'm not at the OCF and I couldn't get my local version of ocfweb to work. I
linked that contrast testing site which shows what the text would look like.

**Comment by edricx** [comment]
I don't understand why the white box was there in the first place, since
none of the other text or elements on the page are wrapped in a white box
(besides one subsection in the section above, although I'm not touching
that at the moment since the alignment would need to be fixed). The design
of the page is centered around having these different colored rows and
every other row has text directly on the colored background. IMO the white
box feels inconsistent with the rest of the sections. The change to black
text addresses the readability issue in my initial edit. Open to hearing
thoughts/reasons on why the white box would be better though.

**Comment by matthew-mcallister** [comment]
The most obvious criticism I can think of is that graphics don't always look that great on colored backgrounds, especially not when they're really close in color to the background color.

**Comment by chriskuehl** [comment]
Yeah, definitely -1 on this change. This looks worse and is much harder to read.

black text on white >>> white text on dark green

**Pull request closed without merging**